README.md

```diff
@@ -785,6 +786,32 @@ in_index = gitlab-mrs_enriched
 out_index = gitlab-mrs-onion_enriched
 data_source = gitlab-merges
 
+```
+#### gitter [&uarr;](#supported-data-sources-)
+Messages from gitter rooms 
+- projects.json
+```
+{
+    "Chaoss": {
+        "gitter": [
+            "https://gitter.im/jenkinsci/jenkins",
+        ]
+    }
+}
+```
+- setup.cfg
+```
+[panels]
+gitter = true
+
+[gitter]
+raw_index = gitter_raw
+enriched_index = gitter_enriched_raw
+api-token = xxxxx
```

Comment:
After `api-token`, please replace the content with
```
sleep-for-rate = true
sleep-time = "300" (optional)
no-archive = true (suggested)
```

The reason is that (i) there are no studies for this data source ATM, (ii) from-date shouldn't be used in a standard execution (since when the from-date is set, all collection tasks start from that date).